docs/rosa/ecr/README.md

```diff
@@ -0,0 +1,79 @@
+# Configuring a ROSA cluster to pull images from AWS Elastic Container Registry (ECR)
+
+## Prerequisites
+
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+* [Openshift CLI](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/) 4.8+
+* [Docker](https://docs.docker.com/get-docker/)
+
+## STS ROSA Clusters
+With STS clusters, you can attach an ECR policy role to your cluster giving the cluster permissions to pull images from your registries.
+
+### Attach ECR Policy Role
```

Comment:
Our usual formatting for steps are like:


### Attach ECR Policy Role

In order to be able to pull images from AWS ECR, you need to assign a policy role.  ROSA comes with a pre-defined policy role `ManagedOpenShift-Worker-Role` which is an IAM role used by ROSA compute instances.

ECR has several pre-defined policies that give permissions to interact with the service.  In the case of ROSA, we will be pulling images from ECR and will only need to add the `AmazonEC2ContainerRegistryReadOnly` policy.  

1. Add the `AmazonEC2ContainerRegistryReadOnly` policy to the `ManagedOpenShift-Worker-Role` 
   ```bash
   aws iam attach-role-policy \
    --role-name ManagedOpenShift-Worker-Role \
    --policy-arn "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
   ```